dev/com.ibm.ws.jcache_fat/publish/servers/jcacheContainerServer/server.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+    <featureManager>
+        <feature>bells-1.0</feature>
```

Comment:
Another variant of this test should involve no bells feature, and package hazelcast jar with the application (or just give the application classloader a commonLibraryRef which will have the same effect).  That can be done in a separate pull